mise.toml

```
@@ -0,0 +1,16 @@
+[tasks.setup]
+description = "Install linting tools from version-pinned files (managed by Dependabot)"
+run = """
+pip install -r .github/requirements.txt
+cd .github && pnpm install --frozen-lockfile
```

```suggestion
pnpm i --frozen-lockfile --dir ./.github
```

Perhaps worth keeping the same command as it was before?